packages/react-reconciler/src/ReactFiberReconciler.js

```diff
@@ -402,7 +403,19 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
     );
   }
 
-  function findHostInstance(fiber: Fiber): PI | null {
+  function findHostInstance(component: Object): PI | null {
```

Comment:
I don't think this is safe because `findHostInstance` is being passed to DevTools below:

https://github.com/facebook/react/blob/76b4ba01290f446f4313adf3846954412c6051b8/packages/react-reconciler/src/ReactFiberReconciler.js#L508-L512

DevTools assumes it can pass a Fiber there.

Reply:
Seems like we should be injecting into devtools from inside the reconciler. Or at least expose something opaque to inject into devtools.

Reply:
Aren't we already? This code is in the reconciler. I'm just pointing out that I think you changed the signature of the function you're passing.

Reply:
ah. I see. Also, spread in production. Sad panda. :(

Reply:
We really should get rid of that. We don't want it in the reconciler so the devtools dep should go away. If it really need it, it can traverse the tree itself, since it is already Fiber aware.

Reply:
Patched it for now.